chat/pom.xml

```diff
@@ -32,6 +32,16 @@
         <artifactId>sakai-chat-impl</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+          <groupId>org.jgroups</groupId>
+          <artifactId>jgroups</artifactId>
+          <version>3.6.13.Final</version>
+      </dependency>
+      <dependency>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
```

Comment:
Mmmmm... as I can see, there is not any guava in shared. Other tools/components using it (like kernel, dashboard or search), are loading it into their component folder.
We can move the library to shared, but then, we need to update all other components. Maybe is better do this in another PR.